src/main/java/techcourse/myblog/repository/UserRepository.java

```diff
@@ -0,0 +1,22 @@
+package techcourse.myblog.repository;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Modifying;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.transaction.annotation.Transactional;
+import techcourse.myblog.model.User;
+
+public interface UserRepository extends JpaRepository<User, Long> {
+    @Query("select u from User u where u.email = ?1")
+    User findUserByEmailAddress(String email);
```

Comment:
아래의 메서드 이름을 사용하면 `@Query`는 필요 없을 거예요.

```suggestion
    User findByEmail(String email);
```

Reply:
확인했습니다!